bindings/nodejs/__test__/fs.spec.mjs

```diff
@@ -0,0 +1,41 @@
+// Copyright 2022 Datafuse Labs
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+import test from 'ava'
+import { Fs } from '../index.js'
```

Comment:
**suggestion**: I expect to see `Fs` and `Memory` exported under a `services` package.

Reply:
😆 Happy to do this.